examples/tests/Functional/LoginCest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tests\Functional;
+
+use Codeception\Attribute\Before;
+use \Tests\Support\FunctionalTester;
```

Comment:
the trailing slash is not necessary, and we should avoid adding it here and anywhere else.